runtime/src/snapshot_utils.rs

```diff
@@ -254,6 +254,9 @@ pub enum SnapshotError {
 
     #[error("snapshot slot deltas are invalid: {0}")]
     VerifySlotDeltas(#[from] VerifySlotDeltasError),
+
+    #[error("snapshot bank verify failed")]
+    BankVerifyFailed,
```

Comment:
The previous error message had the slot in it, would be nice to keep that imo.

Reply:
Thanks!  Sure, will make the change accordingly.